server/controller/signup.js

```diff
@@ -48,7 +48,7 @@ exports.post = async (request, response) => {
               employeeSkillsData = snakeCase(employeeSkillsData);
               await employeeSkills.create(employeeSkillsData);
             });
-            response.status(200).send('Successfule signup, you can login now');
+            response.status(200).send('Successful Signup, You Can Login Now !');
```

Comment:
It’s probably not so important for this project, but in general it’s a more flexible design to have your backend send non-user-facing status text, like result: ‘success’ or result: ‘error’ and then have your front end turn that into the user-facing strong “.. you can login now”. That enables you to internationalize / translate your app to other languages much more easily, as that string replacement logic is usually done all in the front end